packages/trader/src/Modules/Page404/Components/Page404.tsx

```diff
@@ -1,10 +1,9 @@
 import React from 'react';
 import { PageError } from '@deriv/components';
 import { routes, getUrlBase } from '@deriv/shared';
-
 import { localize } from '@deriv/translations';
 
-const Page404 = () => (
+const Page404: React.FC = () => (
```

Comment:
@akmal-deriv there's no need for this  🙂  because 1) React.FC is used with props types passed into it, and we don't have any props to pass here, 2) in view of all the boiling debates around React.FC lately, it's better to not use it without actual reason, e.g. in order to make TS know that this component can return null or does NOT have children.
Opinions among our own devs vary greatly:
Some believe it's a bad practice to use it unless there is no other way, and it's enough to only type props, and based on some official docs and PRs, there were plans to deprecate it. We even have internal doc saying it's discouraged to use it!
And others believe it's ok to use it, it's not gonna be deprecated, and it's not a bad practice.
So let's just avoid it if not absolutely necessary. Is it ok with you?  )

Reply:
Hey @maryia-deriv, sure, I'm not aware of our debates regarding using/not using this and don't have any issues with it, so I added it. I can remove it as well, not crucial at all.

Reply:
Removed it from the file